src/core/dfn.js

```diff
@@ -238,3 +243,32 @@ function addContractDefaults() {
     dfn.dataset.export = "";
   }
 }
+
+// - Sets data-defines on well-known definition content patterns
+function addDefinitionPointers() {
+  // A dl with class hasdefinitions marks all next siblings of dt with the class
+  // definition
+  /** @type NodeListOf<HTMLElement> */
+  const describedDTs = document.querySelectorAll("dl.hasdefinitions dt");
+  for (const dt of describedDTs) {
+    dt.nextElementSibling.classList.add("definition");
+  }
+
+  // an element with class "definition" is marked as defining the term
+  // found either it the element itself, or in the closest previous sibling with
+  // a defined term
+  /** @type NodeListOf<HTMLElement> */
+  const definitionContainers = document.querySelectorAll(".definition");
+  for (const el of definitionContainers) {
```

Comment:
I think we should exclude (or lint) those that already have `data-defines` set to a given local dfn.

Reply:
Could be follow-up.

Reply:
This feels like it should be a CSS query, not a for loop.

Reply:
I've included that in my latest commit